Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -314,6 +319,8 @@ void DoState(PointerWrap& p)
 
   p.Do(s_disc_path_to_insert);
 
+  p.Do(s_rtc_flag);
```

Comment:
Since you are making changes to the savestate system, you need to update the version number in State.cpp.